internal/databases/mongo/archive/layout.go

```diff
@@ -175,7 +175,7 @@ func OldestBackupAfterTime(backups []*models.Backup, after time.Time) (*models.B
 func SelectPurgingOplogArchives(archives []models.Archive,
 	backups []*models.Backup,
 	retainAfterTS *models.Timestamp) []models.Archive {
-	var purgeArchives []models.Archive
```

Comment:
Unit tests are failing because they expect `nil` but get empty slice. Maybe it is better not to refactor this line

Reply:
fixed